src/registrar/models/waffle_sample.py

```diff
@@ -0,0 +1,19 @@
+from waffle.models import AbstractBaseSample
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class WaffleSample(AbstractBaseSample):
+    """
+    Custom implementation of django-waffles 'sample' object.
```

Comment:
**blocking question** Hey why are we adding samples and switches? I specifically called out in the ticket to not add them since we aren't choosing to use them yet. Curious if you found something that made you want to add it now or what?

Reply:
I'm confused are you 1.) saying that we can just delete these and the "register" lines and be fine or 2.) saying you have to have these in and then do something extra so they don't appear.

Either way, we should not be adding code which we have no plan at them moment to ever use. Remove these and the registered portions unless you are actually saying 2 where there is something that breaks when they are removed